client/src/App.js

```diff
@@ -1,16 +1,18 @@
-import React from 'react'
-import { Route, BrowserRouter as Router, Switch } from 'react-router-dom'
-import Start from './components/Start'
-import './App.css'
+import React from "react"
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom"
+import Start from "./components/Start"
+import "./App.css"
+import ListOfItems from "./components/ListOfItems"
 
 const App = () => {
-	return (
-		<Router>
-			<Switch>
-				<Route exact path="/" component={Start} />
-			</Switch>
-		</Router>
-	)
+    return (
+        <Router>
+            <Switch>
+                <Route exact path="/" component={Start} />
+                <Route exact path="/listOfItems" component={ListOfItems} />
```

Comment:
This endpoint is not documented. Are you sure we need the list of items component rendered through `/listOfItems`?